float32 vel_x
float32 vel_y
float32 vel_z
float32 vel_eul_x
float32 vel_eul_y
float32 vel_eul_z